subprojects/core/src/integTest/groovy/org/gradle/normalization/ConfigureRuntimeClasspathNormalizationIntegrationTest.groovy

```diff
@@ -80,6 +85,7 @@ class ConfigureRuntimeClasspathNormalizationIntegrationTest extends AbstractInte
         'nested in dir jars' | 'ignoredResourceInNestedInDirJar' | 'notIgnoredResourceInNestedInDirJar' | Api.ANNOTATION
     }
 
+    @ToBeFixedForConfigurationCache(because = "classpath normalization")
```

Comment:
This always should have been failing with config cache because we use classpath normalization in the test.  But there was an issue with how we were formatting the manifest which was causing it to pass incorrectly in that case.  Fixing the manifest caused the test to fail (as it should) so we need the annotation here.

Reply:
Can we point those tests to https://github.com/gradle/gradle/issues/13706, so it is clear there is an issue on the configuration cache side?

Reply:
Yep - done.